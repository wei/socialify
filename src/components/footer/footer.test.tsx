import { render } from '@testing-library/react'
import Footer from '@/src/components/footer/footer'

jest.mock('@/common/helpers', () => ({
  version: '0.0.0',
}))

test('Footer renders', () => {
  const { container } = render(<Footer />)
  const footer = container.firstElementChild!

  expect(footer).toMatchSnapshot()
})
