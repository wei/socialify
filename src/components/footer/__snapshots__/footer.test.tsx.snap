// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Footer renders 1`] = `
<footer
  class="footer footer-center p-2 text-base-content font-semibold no-screenshot"
>
  <div>
    <p>
      <span>
        Made with ðŸ’– by 
      </span>
      <a
        class="link link-accent no-underline"
        href="https://cryogenicplanet.tech"
        target="_blank"
      >
        CryogenicPlanet
      </a>
      Â andÂ 
      <a
        class="link link-accent no-underline"
        href="https://github.com/wei/"
        target="_blank"
      >
        Wei
      </a>
      <span>
        Â 
        <em>
          (v
          0.0.0
          )
        </em>
      </span>
    </p>
  </div>
  <a
    class="absolute bottom-2 right-2"
    href="https://www.netlify.com"
    target="_blank"
  >
    <img
      alt="Deploys by Netlify"
      height="33"
      src="https://www.netlify.com/v3/img/components/netlify-dark.svg"
      width="75"
    />
  </a>
</footer>
`;
